src/nft/components/bag/BagFooter.tsx

```diff
@@ -454,6 +454,7 @@ export const BagFooter = ({ setModalIsOpen, eventProperties }: BagFooterProps) =
     using_erc20: !!inputCurrency,
     ...eventProperties,
   }
+  const chainAllowed = isSupportedChain(chainId)
 
   return (
     <FooterContainer>
```

Comment:
I think it'd be better to hide the whole section so you don't get this view
![Screen Shot 2023-06-13 at 15 41 26 ](https://github.com/Uniswap/interface/assets/11512321/dd36823c-9715-4a5e-a757-3949aad40078)